src/layouts/AuthLayout.vue

```diff
@@ -1,5 +1,3 @@
 <template>
-  <Suspense>
-    <router-view />
-  </Suspense>
+  <router-view />
```

Comment:
Why remove the Suspense?

Reply:
For some reason this caused the login to fail when returning from signing.